specification.yaml

```diff
@@ -0,0 +1,82 @@
+openapi: 3.0.3
```

Comment:
In the future I hope we can find a system that makes sure the API specification is in synch with the API.

API first approach is one possibility, if there's tooling for it. If the generated code is satisfactory, that is the best option IMO. Code first is another one, again requires tooling to generate a good spec from code.

Reply:
Haven't really fallen in love with the API first approach since I have very strong opinions about project structure, but I haven't really tried it either. 

I'm testing out [this](https://github.com/mpashkovskiy/express-oas-generator) generator in a private project which records the requests sent and received and generates a spec from the result. Maybe that could be used here as well